.github/workflows/publish-commit-bottles.yml

```diff
@@ -23,51 +23,30 @@ jobs:
             })
             const email = (user.data.email || user.data.id + "+" + user.data.login + "@users.noreply.github.com")
             console.log("Dispatched by " + user.data.name + " <" + email + ">")
-            process.stdout.write("::set-output name=name::" + user.data.name + "\n")
-            process.stdout.write("::set-output name=email::" + email + "\n")
+            core.setOutput("name", user.data.name)
+            core.setOutput("email", email)
+      - name: Update Homebrew
+        run: |
+          brew update-reset ${{brew --repository}}
       - name: Checkout tap
         uses: actions/checkout@v2
         with:
           token: ${{secrets.HOMEBREW_GITHUB_API_TOKEN}}
           fetch-depth: 0
       - name: Setup tap
-        env:
-          GIT_COMMITTER_NAME: ${{steps.env.outputs.name}}
-          GIT_COMMITTER_EMAIL: ${{steps.env.outputs.email}}
         run: |
           rm -rf $(brew --repository ${{github.repository}})
           ln -s $GITHUB_WORKSPACE $(brew --repository ${{github.repository}})
-          cd $(brew --repository ${{github.repository}})
-          git fetch origin +refs/pull/${{ github.event.client_payload.pull_request }}/head:current-pr
-          git rebase current-pr master
-          git show
-      - name: Download artifacts
-        uses: Homebrew/actions/download-artifact@master
-        with:
-          github_token: ${{secrets.HOMEBREW_GITHUB_API_TOKEN}}
-          workflow: tests.yml
-          pr: ${{github.event.client_payload.pull_request}}
-          name: bottles
-          path: bottles
-      - name: Upload bottles
+      - name: Pull bottles
         env:
           HOMEBREW_BINTRAY_USER: brewtestbot
           HOMEBREW_BINTRAY_KEY: ${{ secrets.HOMEBREW_BINTRAY_KEY }}
-        run: |
-          cd bottles
-          brew test-bot --ci-upload --publish
-      - name: Push bottle commit
-        env:
           GIT_COMMITTER_NAME: ${{steps.env.outputs.name}}
           GIT_COMMITTER_EMAIL: ${{steps.env.outputs.email}}
         run: |
-          cd $(brew --repo ${{github.repository}})
-          git rebase --force origin/master
-          MESSAGE="$(git log HEAD^.. --format=%B)
-
-          Closes #${{github.event.client_payload.pull_request}}."
-          git commit --amend --signoff --allow-empty -m "$MESSAGE"
-          git show --pretty=fuller
+          brew pr-pull ${{github.event.client_payload.pull_request}}
```

Comment:
Do we need to set `HOMEBREW_GITHUB_API_TOKEN` for `brew pr-pull`?

Reply:
I’m not sure if GITHUB_TOKEN is sufficient. It should need the same secrets as download-artifact I believe?

Reply:
Guess we can merge it now, test it in practice and fix later if needed.